test/CMakeLists.txt

```diff
@@ -69,11 +69,9 @@ endif()
 add_custom_target(build-onedpl-tests
     COMMENT "Build all oneDPL tests")
 
-if (ONEDPL_USE_DR)
-    set(run-onedpl-tests-depends build-onedpl-tests shp-tests shp-tests-3)
-else()
-    set(run-onedpl-tests-depends build-onedpl-tests)
-endif()
+set(run-onedpl-tests-depends build-onedpl-tests)
+
+option(DR_DOCKER "Build shp DR tests with Docker" OFF)
```

Comment:
What is the purpose of this option? It is not set anywhere.

Reply:
it is used not to create new targets (build-dr-tests and run-dr-tests) to be run with others targets; for now they are run in a docker container

Reply:
I still don't see sense of having this DR_DOCKER option. It is a hack. Passing responsibility to the user for determining whether DR tests will compile or not. It should depend on our checks on environment like in https://github.com/oneapi-src/oneDPL/blob/distributed-ranges/test/distributed-ranges/shp/CMakeLists.txt

Reply:
Deleted with unused targets as we discussed